lightkurve/correctors/cbvcorrector.py

```diff
@@ -1,29 +1,735 @@
+        Object detailing the data products found.
+
+    Examples
+    --------
+    This example will read in the CBVs for Kepler quarter 8, 
+    and then extract the first 8 CBVs for module.output 16.4
+
+        >>> cbvs = get_cbvs(mission='Kepler', quarter=8, module=16, output=4,   # doctest: +SKIP
+        >>>     cbv_indices=np.arange(1,9))                                     # doctest: +SKIP
+
+    This example will read in the CBVs for TESS Sector 10 Camera.CCD 2.4
+    and then extract the first 6 CBVs of multi-scale band 2
+
+        >>> cbvs = search_cbvs(mission='TESS', sector=10, camera=2, CCD=4, # doctest: +SKIP
+        >>>     cbv_type='MultiScale', band=2, cbv_indices=np.arange(1,7)) # doctest: +SKIP
+    """
+
+    #***
+    # Validate inputs 
+    # Make sure only the appropriate arguments are passed
+    # TODO: figure out a more elegant way to do this
+    if (mission == 'Kepler'):
+        assert  isinstance(quarter, int), 'quarter must be passed for Kepler mission'
+        # All these inputs are invalid
+        assert  campaign is None, 'campaign must not be passed for Kepler mission'
+        assert  sector is None,   'sector must not be passed for Kepler mission'
+        assert  camera is None,   'camera must not be passed for Kepler mission'
+        assert  CCD is None,      'CCD must not be passed for Kepler mission'
+    elif (mission == 'K2'):
+        assert  isinstance(campaign, int), 'campaign must be passed for K2 mission'
+        # All these inputs are invalid
+        assert  quarter is None,  'quarter must not be passed for K2 mission'
+        assert  sector is None,   'sector must not be passed for K2 mission'
+        assert  camera is None,   'camera must not be passed for K2 mission'
+        assert  CCD is None,      'CCD must not be passed for K2 mission'
+    elif (mission == 'TESS'):
+        assert  isinstance(sector, int),    'sector must be passed for TESS mission'
+        assert  not camera is None,   'camera must be passed'
+        assert  not CCD is None,      'CCD must be passed'
+        # All these inputs are invalid
+        assert  quarter is  None,  'quarter must not be passed for TESS mission'
+        assert  campaign is None, 'campaign must not be passed for TESS mission'
+    else:
+        raise Exception('Unknown mission type')
+        
+    try: 
+
+        
+
+        if (mission == 'Kepler' or mission == 'K2'): 
+
+            # CBV FITS files use module/output, not channel
+            # So if channel is passed, convert to module/output
+            if (isinstance(channel, int)):
+                assert  module is None, 'module must NOT be passed if channel is passed'
+                assert  output is None, 'output must NOT be passed if channel is passed'
+                module, output = channel_to_module_output(channel)
+                channel = None
+            else:
+                assert  not module is None, 'module must be passed'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [test for object identity should be 'is not'](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4896575)